packages/react-internal/src/components/Callout/Callout.test.tsx

```diff
@@ -51,7 +51,16 @@ describe('Callout', () => {
     Component: Callout,
     displayName: 'Callout',
     targetComponent: CalloutContent,
-    disabledTests: ['component-handles-ref'],
+    // Disabled Tests:
+    //
+    // component-handles-ref
+    // Problem: Ref isn't receiving the correct element.
+    // Solution: Investigate the refs implementation.
+    //
+    // consistent-callback-names
+    // Problem: Contains onLayerMounted and onPositioned types.
+    // Solution: Remove the inconsistent callback names.
+    disabledTests: ['component-handles-ref', 'consistent-callback-names'],
```

Comment:
Instead of disabling this, it would probably be better to add `testOptions['consistent-callback-names'].ignoreProps` (same in other cases too)